### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - David Berger, Sara Camassa

### DISCUSSION THREAD

**Discussion by scamassa** [heig-vd-dai-course/heig-vd-dai-course#534] [Show and tell]
For this practical work, we would like to manage two "databases" :

- a pokedex
->  filled with some existing pokemon
-> display the pokemon by type, id, etc
-> add pokemon in the pokedex

- a pc
-> only add pokemon that exists in the pokedex
-> update the pokemon (example: change the name)
-> display the pokemon by level, type, ect
-> delete a pokemon

repo: [https://github.com/scamassa/DAI_Lab03](https://github.com/scamassa/DAI_Lab03)
last commit : d7a824002e01cf706de9695081dc8b8535de40e5

**Comment by GeraudSilvestri** [comment]
Good for me
